src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,128 @@
+import classNames from 'classnames';
+import React, { FormEvent, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  deleteTodo: (id: number) => void,
+  processedIds: number[],
+  handleCheckbox: (id: number, value: boolean) => void,
+  handleChangeTitle: (id: number, value: string) => void,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  deleteTodo,
+  processedIds,
+  handleCheckbox,
+  handleChangeTitle,
+}) => {
+  const { id, completed, title } = todo;
+
+  const [isEditing, setIsEditing] = useState(false);
+  const [titleInput, setTitleInput] = useState(title);
+
+  const handleSubmitOrBlur = (event: FormEvent) => {
+    event.preventDefault();
+
+    if (titleInput === '') {
```

Comment:
If the mentor leaves similar comments (there has already been one about the length of the todo array), then you need to find all similar places in your code and correct them
```suggestion
    if (!titleInput) {
```